local/client/conn/conn.go

```diff
@@ -19,6 +20,12 @@ type Dialer interface {
 
 type Conn interface {
 	Echo(arg string) (string, error)
+
+	// TODO(dbentley): this feels weird. We shouldn't expose our internal
```

Comment:
As long as it's isolated so you can pull in exactly the dependencies you need, I don't see a huge problem with libs referring to higher level dirs.